pom.xml

```diff
@@ -36,39 +36,39 @@
 	</modules>
 
 	<properties>
-        <openMRSVersion>1.9.11</openMRSVersion>
-        <addresshierarchyVersion>2.10.0</addresshierarchyVersion>
-        <appframeworkVersion>2.10.0</appframeworkVersion>
-        <appuiVersion>1.8.0</appuiVersion>
-        <calculationVersion>1.2</calculationVersion>
-		<coreappsVersion>1.16.0</coreappsVersion>
-        <emrapiVersion>1.21.0</emrapiVersion>
-        <eventVersion>2.5</eventVersion>
-        <htmlformentryuiVersion>1.12.0-SNAPSHOT</htmlformentryuiVersion>
-        <htmlformentryVersion>3.9.2</htmlformentryVersion>
-        <htmlwidgetsVersion>1.8.0</htmlwidgetsVersion>
-        <idgenVersion>4.5.0</idgenVersion>
-        <groovyVersion>2.2.4</groovyVersion>
-        <logicVersion>0.5.2</logicVersion>
-        <metadatadeployVersion>1.11.0</metadatadeployVersion>
-        <metadatamappingVersion>1.3.1</metadatamappingVersion>
-        <metadatasharingVersion>1.2.2</metadatasharingVersion>
-        <namephoneticsVersion>1.5</namephoneticsVersion>
-        <providermanagementVersion>2.10.0</providermanagementVersion>
-        <reportingVersion>1.18.0</reportingVersion>
-        <reportinguiVersion>1.2</reportinguiVersion>
-        <reportingrestVersion>1.7.0</reportingrestVersion>
-        <reportingcompatibilityVersion>1.5.8</reportingcompatibilityVersion>
-        <serializationVersion>0.2.12</serializationVersion>
-        <registrationcoreVersion>1.6</registrationcoreVersion>
-        <registrationappVersion>1.7.0</registrationappVersion>
-        <syncVersion>1.3</syncVersion>
-        <testutilsVersion>1.5</testutilsVersion>
-        <uicommonsVersion>2.3.0</uicommonsVersion>
-        <uiframeworkVersion>3.11.0</uiframeworkVersion>
-        <uilibraryVersion>2.0.4</uilibraryVersion>
-        <validationVersion>1.0.1</validationVersion>
-        <webservicesRestVersion>2.25.0</webservicesRestVersion>
+        <openMRSVersion>2.3.4-SNAPSHOT</openMRSVersion>
+        <addresshierarchyVersion>2.14.2</addresshierarchyVersion>
+        <allergyuiVersion>1.8.3</allergyuiVersion>
+        <appframeworkVersion>2.16.0</appframeworkVersion>
+        <appuiVersion>1.14.0</appuiVersion>
+        <calculationVersion>1.3.0</calculationVersion>
+        <coreappsVersion>1.33.0</coreappsVersion>
+        <emrapiVersion>1.33.0-SNAPSHOT</emrapiVersion>
```

Comment:
are we installing the full refapp here?  do we need coreapps, emrapi, appui, etc?

Reply:
coreapps was already in the pihmalawi distro. It is needed for providermanagement a few other features that require the UI framework.